packages/back-end/src/api/organizations/listEnvironments.ts

```diff
@@ -0,0 +1,28 @@
+import { ListEnvironmentsResponse } from "../../../types/openapi";
+import { createApiRequestHandler } from "../../util/handler";
+import { findOrganizationById } from "../../models/OrganizationModel";
+import { listEnvironmentsValidator } from "../../validators/openapi";
+
+export const listEnvironments = createApiRequestHandler(
+  listEnvironmentsValidator
+)(
+  async (req): Promise<ListEnvironmentsResponse> => {
+    const id = req.params.id;
+
+    const org = await findOrganizationById(id);
+    if (!org) {
+      throw Error("Organization not found");
+    }
```

Comment:
Same as the other endpoints.  This one is especially unsafe because anyone can list any other org's environments even without being part of the org.